frontend/modelarchive/src/main/java/org/pytorch/serve/archive/ModelArchive.java

```diff
@@ -133,14 +133,10 @@ public static File unzip(InputStream is, String eTag) throws IOException {
         }
         ZipUtils.unzip(new DigestInputStream(is, md), tmp);
         if (eTag == null) {
-            eTag = HexUtils.toHexString(md.digest());
+            eTag = UUID.randomUUID().toString().replaceAll("-", "");
         }
+        logger.info("eTag {}", eTag);
```

Comment:
NIT: log message is not descriptive. We should probably add the model name to go along with the etag

Reply:
I printed the model name and this is the log.

TorchServeSuite > TorchServe > org.pytorch.serve.ModelServerTest > testLoadModelWithInitialWorkersWithJSONReqBody STANDARD_OUT
    2020-08-04 01:41:03,788 [DEBUG] nioEventLoopGroup-3-4 org.pytorch.serve.wlm.ModelVersionedRefs - Removed model: noop version: 1.11
    2020-08-04 01:41:03,794 [INFO ] nioEventLoopGroup-3-4 org.pytorch.serve.wlm.ModelManager - Model noop unregistered.
    2020-08-04 01:41:03,795 [INFO ] nioEventLoopGroup-3-4 ACCESS_LOG - /127.0.0.1:63658 "DELETE /models/noop HTTP/1.1" 200 7
    2020-08-04 01:41:03,795 [INFO ] nioEventLoopGroup-3-4 TS_METRICS - Requests2XX.Count:1|#Level:Host|#hostname:czhong03,timestamp:null
    2020-08-04 01:41:03,799 [INFO ] nioEventLoopGroup-3-4 org.pytorch.serve.wlm.ModelManager - modelName : noop
    2020-08-04 01:41:03,799 [INFO ] nioEventLoopGroup-3-4 org.pytorch.serve.archive.ModelArchive - marFileName : noop.mar
    2020-08-04 01:41:03,805 [INFO ] nioEventLoopGroup-3-4 org.pytorch.serve.archive.ModelArchive - model folder already exists: 29385dfc880480adb4ff8a26d9461d72539fb887
    2020-08-04 01:41:03,806 [INFO ] nioEventLoopGroup-3-4 ACCESS_LOG - /127.0.0.1:63658 "POST /models HTTP/1.1" 400 8
    2020-08-04 01:41:03,806 [INFO ] nioEventLoopGroup-3-4 TS_METRICS - Requests4XX.Count:1|#Level:Host|#hostname:czhong03,timestamp:null

TorchServeSuite > TorchServe > org.pytorch.serve.ModelServerTest > testLoadModelWithInitialWorkersWithJSONReqBody FAILED
    java.lang.AssertionError at ModelServerTest.java:317

Reply:
```
./torchserve_sanity.sh | grep --line-buffered eTag
+ source scripts/install_utils
++ set -euo pipefail
+ set +u
+ install_torch_deps
+ '[' is_gpu_instance ']'
+ '[' '' == cuda101 ']'
+ pip install -U -r requirements.txt
+ set -u
+ install_pytest_suite_deps
+ pip install -U -r requirements/developer.txt
+ install_bert_dependencies
+ pip install transformers
+ run_backend_pytest
+ python -m pytest --cov-report html:htmlcov --cov=ts/ ts/tests/unit_tests/
+ echo 'Backend test suite execution successful'
+ build_frontend
+ frontend/gradlew -p frontend clean build
    2020-08-04 17:38:35,334 [INFO ] Test worker org.pytorch.serve.archive.ModelArchive - eTag 8652c3fcf1494865a7fb5126afc74e4a
    2020-08-04 17:38:36,527 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag cd453c8393b2476db54af1f3d544ec39
    2020-08-04 17:38:36,650 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag a9cf7eae14db4aa99da3e3238287adb6
    2020-08-04 17:38:36,748 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag 254257fef1704f879291a0383bf132e0
    2020-08-04 17:38:36,844 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag fbd18c2405d24ea18e188f4ffef89909
    2020-08-04 17:38:36,971 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag eb7bba80a9394835b36a31d0e70fe5bc
    2020-08-04 17:38:37,080 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag fd300918ceb14fddbc29358aad5db14d
    2020-08-04 17:38:37,180 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag 3d5f0fcf085e4934bed16e8bda5f6b7c
    2020-08-04 17:38:37,975 [INFO ] epollEventLoopGroup-3-2 org.pytorch.serve.archive.ModelArchive - eTag 23c3f2c81fd74e13b698e786be192b92
```

Reply:
> NIT: log message is not descriptive. We should probably add the model name to go along with the etag

`model name` doesn't passed to function ModelArchive.unzip() nor function ModelArchive.downloadModel(String modelStore, String url), so `add the model name to go along with the etag` need to change all the functions that us them. And use UUID is an easier and better way to distinguish these files. 

When I run `frontend/gradlew.bat -p frontend clean build` on Windows 10, there are 3 test cases using the same eTag 29385dfc880480adb4ff8a26d9461d72539fb887, here is the log https://paste.ubuntu.com/p/sBjJmnWDSv/